btr-api/src/btr_api/services/request.py

```diff
@@ -90,45 +90,28 @@ def update_request(request: RequestModel,
         - RequestModel: A RequestModel object that represents the updated request
         """
 
-        # I dislike that I can't use a for loop here but it doesn't like to access the model with [key]
-        # even if i override __setitems__ or add a set function
-        if 'fullName' in request_dict:
-            request.full_name = request_dict['fullName']
-
-        if 'email' in request_dict:
-            request.email = request_dict['email']
-
-        if 'birthdate' in request_dict:
-            request.birthdate = request_dict['birthdate']
-
-        if 'businessIdentifier' in request_dict:
-            request.business_identifier = request_dict['businessIdentifier']
-
-        if 'informationToOmit' in request_dict:
-            request.information_to_omit = request_dict['informationToOmit']
-
-        if 'individualAtRisk' in request_dict:
-            request.individual_at_risk = request_dict['individualAtRisk']
-
-        if 'reasons' in request_dict:
-            request.reasons = request_dict['reasons']
-
-        if 'completingParty' in request_dict:
-            request.completing_party = request_dict['completingParty']
-
-        if 'completingEmail' in request_dict:
-            request.completing_email = request_dict['completingEmail']
-
-        if 'completingName' in request_dict:
-            request.completing_name = request_dict['completingName']
-
-        if 'status' in request_dict:
-            request.status = request_dict['status']
-
-        # for key in dict_keys:
-        #     if key in request_dict:
-        #         request[key] = request_dict[key]
-
-        request['updatedAt'] = datetime.today().strftime('%Y-%m-%dT%H:%M:%S')
+        field_mappings = {
+            'fullName': 'full_name',
+            'email': 'email',
+            'birthdate': 'birthdate',
+            'informationToOmit': 'information_to_omit',
+            'individualAtRisk': 'individual_at_risk',
+            'reasons': 'reasons',
+            'completingParty': 'completing_party',
+            'completingEmail': 'completing_email',
+            'completingName': 'completing_name',
+            'status': 'status'
+        }
+
+        # Update standard fields using mapping
+        for dict_key, model_attr in field_mappings.items():
+            if dict_key in request_dict:
+                setattr(request, model_attr, request_dict[dict_key])
+
+        # Handle special case for businessIdentifiers (requires joining)
+        if 'businessIdentifiers' in request_dict:
```

Comment:
I think this should be removed now,I suspect if it was in  the update test it would fail